source/connect.txt

```diff
@@ -90,16 +90,32 @@ Steps
       .. image:: /images/connect/intellij-launch-playgrounds.png
          :alt: To launch the Playgrounds, click the highlighted button in Database Explorer.
 
+   .. step:: (*Optional*) Select a data source using the Connections Toolbar.
+
+      The Connections Toolbar is a feature of the {+intellij-full+} that 
+      allows you to select a data source associated with your application
+      from within the editor window.
+      
+      When you open a file that contains MongoDB code, a toolbar appears at 
+      the top of the file as shown in the image below. The Connections 
+      Toolbar is highlighted in red.
```

Comment:
[suggestion] Since the previous sentence points out the toolbar being in the top of the file as shown in the image, we don't need to specifically point out that it's highlighted in red.

Reply:
Updated!